docs/index.md

```diff
@@ -51,11 +51,11 @@ FuzzBench, submit it, and get benchmark results.
 
 ## Sample report
```

Comment:
I think we need to change this section name.
Also, while I think it's necessary to link to the list of reports, I don't think we should get rid of the link to the sample. 
The sample shows what fuzzbench is capable of.
Showing people the list of reports may not demonstrate this since the latest report might be for a currently running experiment and/or may not contain all fuzzers/benchmarks.